docs/usage/registry-cache/monitoring.md

```diff
@@ -0,0 +1,72 @@
+# Monitoring
+
+The `registry-cache` extension exposes metrics for the [distribution](https://github.com/distribution/distribution) `registries` running on a Shoot cluster as `StatefulSets` so that they can be easily viewed by cluster owners and operators in the Shoot's Prometheus and Plutono instances. The exposed monitoring data provides an overview of the performance of the `registry` pull-through caches, including hit rate and traffic to and from the `registry`.
```

Comment:
```suggestion
The `registry-cache` extension exposes metrics for the registry caches running in the Shoot cluster so that they can be easily viewed by cluster owners and operators in the Shoot's Prometheus and Plutono instances. The exposed monitoring data provides an overview of the performance of the pull-through caches, including hit rate and network traffic data.
```

- There is no need to reveal implementation detail that we run registries as StatefulSets.
- For consistency reasons, use "registry cache" or "pull-through cache" for naming the cache that is deployed in the Shoot - not "registry" or "proxy".